src/Appstream/AppstreamClient.php

```diff
@@ -42,4 +42,16 @@
  * @method \Aws\Result updateStack(array $args = [])
  * @method \GuzzleHttp\Promise\Promise updateStackAsync(array $args = [])
  */
-class AppstreamClient extends AwsClient {}
+class AppstreamClient extends AwsClient {
+
+    public function __construct(array $args)
+    {
+        if (!isset($args['endpoint'])) {
```

Comment:
@jeskew Actually... perhaps not, when I try to change service identifier, it will fail at resolving api models:
```
Aws\Exception\UnresolvedApiException: The appstream2 service does not have version: ..
```
This is because their api model haven't been renamed to `appstream2` ...